PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,36 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
```

Comment:
I notice this points to a `awsstg` URL, but customer can't reach that can they?

Reply:
Good questions, but I'm not sure the answer. @asinghjc could you help out here?

Reply:
Hi Scott and Wes,
The awsstg link for downloading the installable files (jumpcloud-assist-app.dmg) are supposed be a public URL. The download links are generated by deploy team with this requirement.
I tried downloading the file on my personal machine using this link and it worked fine.

Thanks,
Ajay